api/endpoint-tests/utils.js

```diff
@@ -3,11 +3,10 @@ const FormData = require('form-data');
 const knex = require('knex');
 const knexConfig = require('../knexfile');
 
-const baseURL = () => `http://${
-    process.env.API_HOST || 'localhost'
-  }:${
-    process.env.API_PORT || process.env.PORT || 8000
-  }`;
+const baseURL = () =>
+  `http://${process.env.API_HOST || 'localhost'}:${process.env.API_PORT ||
+    process.env.PORT ||
+    8000}`;
```

Comment:
I had this formatted the other way so that each part of the URL in question was on it's own line. Seems like pretter/eslint prefers this formatting... 🤷 

No change necessary.